src/main/java/eu/europeana/iiif/model/v2/ManifestV2.java

```diff
@@ -12,12 +12,13 @@
  * @author Patrick Ehlert
  * Created on 06-02-2018
  */
-@JsonPropertyOrder({"id", "context"})
+@JsonPropertyOrder({"id", "service", "context"})
```

Comment:
This should probably read {"id", "service", "@ context"} (without a space beween @ and context, but that was added to avoid refering to a git-user named context).  However I don't like that order.
I just talked to the PM and he prefers to keep the same order defined in his EDM mapping document. However that does not have service yet, so I asked him to add this.

Reply:
Remarks now implemented.